content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -129,6 +129,7 @@ For a reference to old feature gates that are removed, please refer to
 | `KMSv2KDF` | `false` | Beta | 1.28 | |
 | `KubeProxyDrainingTerminatingNodes` | `false` | Alpha | 1.28 | |
 | `KubeletCgroupDriverFromCRI` | `false` | Alpha | 1.28 | |
+| `KubeletEnsureSecretPulledImages` | `false` | Alpha | 1.29 | |
```

Comment:
Each and every gate needs a description that can be found at the end of this file, in alphabeta order.